docs/baking.md

```diff
@@ -123,50 +128,10 @@ sudo -u tezos tezos-node config init --data-dir /var/lib/tezos/node-<network> --
 sudo -u tezos tezos-node config update --data-dir /var/lib/tezos/node-<network> --network <network>
 ```
 
-### Starting the node
-
-After the snapshot import, you can finally start the node by running:
-```
-sudo systemctl start tezos-node-<network>.service
-```
-
-Note that even after the snapshot import the node can still be out of sync. It may require
-some additional time to completely bootstrap. In order to check whether the node is bootstrapped,
-you can use `tezos-client`:
-```
-sudo -u tezos tezos-client bootstrapped
-```
-
-To stop the node, run:
-```
-sudo systemctl stop tezos-node-<network>.service
-```
-
-If you want the service to start automatically at boot, use `enable`:
-```
-sudo systemctl enable tezos-node-edo2net.service
-```
-and to disable it from doing so, instead use:
-```
-sudo systemctl disable tezos-node-edo2net.service
-```
-
-You can check node logs via `journalctl`:
-```
-journalctl -u tezos-node-<network>.service
-```
-
-## Setting up baker and endorser daemons
-
-### Setting up daemon data directories
-
-Data directories for baker and endorser daemons are defined in the
-`/etc/default/tezos-baker-<proto>` and `/etc/default/tezos-endorser-<proto>`.
-By default, both these daemons have `/var/lib/tezos/.tezos-client` set as a `DATA_DIR`.
-
-Additionally, you need to specify `NODE_DATA_DIR` in the `/etc/default/tezos-baker-<proto>`
-to point at the desired node data directory, e.g. `/var/lib/tezos/node-<network>`.
+## Setting up baker and endorser key
 
+By default `tezos-baking-<network>.service` will use `baker` alias for the
+key that will be used for baking and endorsing.
 
 ### Importing the baker key
```

Comment:
Also, I think we can move `Registering the baker` section to the very end after `Starting baking instance`

Reply:
> There's a problem here now, as the node hasn't started yet the tezos-client commands below will fail.

Hmm, indeed, though, only `tezos-client activate account` will fail in case there is no node, the rest of imports' commands will just show a warning